File: python/src/tests/t_create.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

key = int(sys.argv[1]);

print('create : ' + str(key) + ' : ')

count = int(sys.argv[2]);

print('create : ' + str(count) + ' : ')

id = atrshmlog.create(key, count)

print('create : ' + str(id) + ' : ')

if id > 0 :
    f = open('dot.atrshmlog', 'w')
    
    c =  'ATRSHMLOG_ID=' + str(id) + '\nexport ATRSHMLOG_ID\nATRSHMLOG_COUNT=' + str(count) + '\nexport ATRSHMLOG_COUNT\n'
    
    f.write(c)
    
    f.close()


print (' ')

exit(0)

# end of test

File: perl/src/tests/t_create.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$key = shift @ARGV;

print "create : $key : \n";

$count = shift @ARGV;

print "create : $count : \n";

$id = Atrshmlog::create(4812,$count);

print "create : $id : \n";

if ($id >  0) {
    open(OUT, ">dot.atrshmlog") or die "cannot open for write\n";

    print OUT "ATRSHMLOG_ID=$id\nexport ATRSHMLOG_ID\nATRSHMLOG_COUNT=$count\nexport ATRSHMLOG_COUNT\n";

    close OUT;
}

print " \n";

exit (0);

# end of main
